catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -517,6 +526,112 @@ public void init() {
             return "";
         });
 
+        post("/annotations", (req, res) -> {
+            Map<String, Object> incoming = JsonFactory.create()
+                    .parser()
+                    .parseMap(req.body());
+            String queryId = incoming.get("parent")
+                    .toString();
+            String annotation = incoming.get("note")
+                    .toString();
+            String user = getSubjectEmail();
+            if (user == null) {
+                return util.getResponseWrapper("error",
+                        "You are not authorized to create notes! A user email is required. "
+                                + "Please ensure you are logged in and/or have a valid email registered in the system.");
+            }
+            if (StringUtils.isBlank(annotation)) {
+                return util.getResponseWrapper("error", "No annotation!");
+            }
+            Metacard note = saveMetacard(new NoteMetacard(noteMetacardType,
```

Comment:
Should we be auditing when a user attaches a note to a resource that they are not the POC for?

Reply:
@bdeining  and I don't think there is a reliable way to detect that the user attaching the note is not the POC, so I've added a SecurityLogger.auditWarn every time an annotation is created.